jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -1070,6 +1070,14 @@ properties:
                 title: Dark reference file name
                 type: string
                 fits_keyword: R_DARK
+          darkMIRI:
```

Comment:
This addition to the core schema is not necessary. It's only the definition of the keyword that will be used to record the name of the dark reference file that was used during calibration. The same keyword "R_DARK" can and should be used for all instruments, hence there's no need for a separate meta data entry.